src/components/pages/doc/chat-widget/chat-widget.jsx

```diff
@@ -0,0 +1,352 @@
+'use client';
+
+import * as Dialog from '@radix-ui/react-dialog';
+import clsx from 'clsx';
+import { AnimatePresence, LazyMotion, domAnimation, m } from 'framer-motion';
+import PropTypes from 'prop-types';
+import { useCallback, useContext, useEffect, useRef } from 'react';
+
+import { ChatContext } from 'app/chat-provider';
+import Button from 'components/shared/button/button';
+import useAbortController from 'hooks/use-abort-controller';
+import useControlKey from 'hooks/use-control-key';
+import useDocsAIChatStream from 'hooks/use-docs-ai-chat-stream';
+
+import AttentionIcon from './images/attention.inline.svg';
+import CloseIcon from './images/close.inline.svg';
+import ExampleIcon from './images/example.inline.svg';
+import ExperimentalIcon from './images/experimental.inline.svg';
+import ReloadIcon from './images/reload.inline.svg';
+import SendIcon from './images/send.inline.svg';
+import SparksIcon from './images/sparks.inline.svg';
+import Message from './message';
+
+const items = [
+  'What’s Neon?',
+  'How to sign up for Neon?',
```

Comment:
This prepared question returns an "I don't know" answer, but if you type in the same question, you get an answer, but it's about the Terms of Service. If you change the question to "How do I sign up for Neon?", you get a far better answer. Can we change the question and test the result?

Reply:
@danieltprice done!